alot/buffers/thread.py

```diff
@@ -297,6 +294,35 @@ def focus_prev_unfolded(self):
         self.focus_property(lambda x: not x.is_collapsed(x.root),
                             self._tree.prev_position)
 
+    def focus_next_thread(self):
+        """focus next thread of the search result"""
+        self._update_thread('down')
+
+    def focus_prev_thread(self):
+        """focus previous thread of the search result"""
+        self._update_thread('up')
+
+    def _update_thread(self, movement):
+        if not self._search_buffer or not self._search_buffer():
+            return
+        search_buffer = self._search_buffer()
+        search_buffer.listbox.keypress((1, 1), movement)
+        new_thread = search_buffer.get_selected_thread()
+        if new_thread is not self.thread:
+            self.thread = new_thread
+            self.body = None
+            self.rebuild()
+            self.ui.update()
+            self.unfold_matching(search_buffer.querystring)
+
+    def focus_next(self):
```

Comment:
why did you move this here?

Reply:
This was a mistake, I will put it back where it was.